dapp/src/components/shared/Tooltip.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from "react"
+import { Box, Tooltip as ChakraTooltip, TooltipProps } from "@chakra-ui/react"
+
+export default function Tooltip(props: TooltipProps) {
+  const { children, ...restProps } = props
+  const [isOpen, setIsOpen] = useState(false)
+
+  return (
+    <ChakraTooltip isOpen={isOpen} {...restProps}>
+      <Box
+        onMouseEnter={() => setIsOpen(true)}
+        onMouseLeave={() => setIsOpen(false)}
+        onClick={() => setIsOpen(true)}
+      >
```

Comment:
Why do we need this?

Reply:
Yeah, I added this to make tooltips available on mobile devices. More info [here](https://github.com/chakra-ui/chakra-ui/issues/2691#issuecomment-760968823).

Reply:
It's worth adding a comment in the code to document it if it's unclear.